### DISCUSSION TITLE
Hello, may I please ask where the application settings are stored?

### DISCUSSION THREAD

**Discussion by vladimir-kordas** [martinet101/ElevenClock#511] [Q&A] [answered]
When I uninstall and reinstall, the old settings are still there.

**Comment by marticliment** [comment, marked as answer]
On a folder called .elevenclock placed in your user directory. Individual settings are stored as files

**Comment by marticliment** [comment]
If the file exists, the setting is enabled (and it might have a value in the file)

If the file does not exist, the setting is set to false (disabled)

**Reply by vladimir-kordas** [reply]
Thanks :)
